backend/middleware/EventValidation.js

```
@@ -0,0 +1,109 @@
+            errors.push('Event name must be between 3 and 100 characters');
+        }
+
+        // Description validation
+        if (!description || description.trim().length < 20 || description.trim().length > 1000) {
+            errors.push('Description must be between 20 and 1000 characters');
+        }
+
+        // Poster URL validation
+        const urlPattern = new RegExp(
+            '^https?:\\/\\/.+\\..+'
+        );
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)